### ISSUE TITLE
Component:  Masthead:  Accessibility - Voiceover iPhone - IBM logo link, screen reader is announcing the mentioned control as "slash link "

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#2055]
## Detailed description

Issue :- The voice over on iPhone is announcing IBM logo link as "slash link"

Expected :- screen reader should announce appropriate label/ link text.

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/f07ad3ef-eb2b-429a-ad98-0b80ec54af6f)

**Comment by jeffchew** [comment]
@Praveen-sr @chetankan can you add severity to these issues? High = `Severity 2`, Medium = `Severity 3`, etc.

**Comment by Praveen-sr** [comment]
@jeffchew Sure thing, will update them.

**Comment by kennylam** [comment]
@Praveen-sr @chetankan @RobertaJHahn 
https://github.com/carbon-design-system/ibm-dotcom-library/pull/2074

The deploy links are in the comments of this PR.
